v1/util/index.js

```diff
@@ -42,9 +42,11 @@ const formatValue = value => {
 // Format the destination.Config.dynamicMap arrays to hashMap
 const getHashFromArray = (arrays, fromKey = "from", toKey = "to") => {
   const hashMap = {};
-  arrays.forEach(array => {
-    hashMap[array[fromKey]] = array[toKey];
-  });
+  if (arrays) {
```

Comment:
let's do Array.isArray(arrays) here?
it wlll then safeguard objects too

Reply:
Done